README.md

```diff
@@ -31,12 +31,13 @@ Once `vagrant up` has finished, follow these steps to start the server.
 
 1. `vagrant ssh` to login into the virtual machine.
 2. `cd /vagrant` to navigate to the project.
-3. `node .` to start the server.
-4. Open your browser
-5. 
+3. `npm run api` to start the api.
+4. `npm run client` to start the client.
```

Comment:
I only show webpack per se running when I use `node .` or `npm run api`, not webpack-dev-server, so file changes are not resulting in browser reloads.